nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/AbstractPutHBase.java

```diff
@@ -63,6 +64,17 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    protected static final PropertyDescriptor DEFAULT_VISIBILITY_STRING = new PropertyDescriptor.Builder()
+            .name("hbase-default-vis-string")
+            .displayName("Default Visibility String")
+            .description("When using visibility labels, any value set in this field will be applied to all cells that are written unless " +
```

Comment:
It might be worth mentioning that the HBase installation that you're trying to write these too must have the VisibilityCoprocessor already loaded. Unsuspecting users might just assume that nifi automagically did something to HBase to let it handle the labels.

Reply:
That's a very good point.